### DISCUSSION TITLE
Merge Queue not closing PRs

### DISCUSSION THREAD

**Discussion by alex-statsig** [community/community#16399] [Pull Requests]
As the title says, when using merge queue occasionally the system does not automatically close the PR after merging. This often leads to our team trying to re-add it to the merge queue, which fails (due to no changes), and blocks the queue.

|Correct behavior|Broken behavior|
|--|--|
|![Screen Shot 2022-05-11 at 1 46 26 PM](https://user-images.githubusercontent.com/77301670/167944797-a6a95248-91a5-4d1c-af8d-e660c5812900.png)|![Screen Shot 2022-05-11 at 1 46 18 PM](https://user-images.githubusercontent.com/77301670/167944802-a096ccd9-7f3e-4b5c-b903-766c81c57284.png)|

**Comment by csantos1113** [comment]
I have encountered this error in the past as well. PRs left with `0 file` changes.

Also, maybe related, I started getting this error behavior as well: https://github.com/github-community/community/discussions/18497
